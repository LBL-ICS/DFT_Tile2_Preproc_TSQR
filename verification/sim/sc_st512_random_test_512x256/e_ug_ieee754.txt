3EF84284
BF7FA607
BD8698D2
3ED9B93A
BEC2CD22
3F5B7FC9
3E5C71F5
C000A15A
3F2A3B94
BE14E7DD
3EFA2A07
3F5FCFCF
3F17C31D
BF7E8489
BF398323
3F080420
3F381EFA
3E49C750
BFA04733
BE8C0793
BF06E06D
3EFD4D55
3EB7CFC9
3F42A2B0
BF9A006B
BF84F858
401CAB6D
BE1F9FA3
3FE7C066
BFD280DA
3FA44CAC
BF99522E
BEB05B7A
3E59CEE7
BE47C8CA
3ECF9AB1
BE9CD44D
BF96C849
3FA96D93
BF2C380A
BE1D975D
BF0D0FA5
3FA00CBF
3FA6FF54
BFDED065
3F7A7DCE
BE48922E
40046C53
BEECC655
3EFF82D6
3F3A0D16
3FCD0A51
3ECF0E86
BED6A7E0
BF8E3779
3FAB770B
BF9C6158
C0073721
401AD911
3F1C2CA4
3FA945FF
C01836B7
BD6DAFD3
3ED2FB39
3FA42E8C
BEB37A1A
3E88823E
3ED387F4
3FDEDA27
3F0F3F4E
3EA3E278
3F27352D
BF8C1BF7
3F72DEC3
BF63E8EE
3D1FFE9C
BEEB9B82
3F78F239
BE9799EB
BF3AE5C3
3DF0983E
3FBB9652
3FC5D7E6
3EFAE123
BE936166
3F89D13A
BE8945AB
3EBF1708
3E7F4719
BFA681A3
3FBEE7E8
BEFCCD8C
BF117CDB
3E2543E6
BF933171
3F8BEA82
BEA4A2FA
BF510EBB
BFADD55A
BEE9C253
3E483ED9
BE96D3E8
3EB6B6CB
3E797F0A
BF5AF6DC
BF0D679E
3E874BB4
BF8A8A3B
BF4FF6B2
BF6FA379
BE75793C
3FC40FD8
3F3D6057
BE0D4E8B
BDFA4DDB
3FCDB92C
BFE484A1
BDE2061A
3F125C0E
3EA57439
BEABCA2F
BF9A54A2
BD348BA0
3F7C0C3D
BF4AA967
BF95321F
BF141E82
BF5B3808
3D176515
BF830E53
3F479527
3FE3A32A
C0159DF7
BEBDE0DD
3F9001BB
3E924DEE
3F82694A
3FAA0E33
3E965051
BE89CF8E
BE87229E
3E303EA5
3F92ABB8
3E7EA1F6
BF054E06
BECA2B80
BF087FAF
C0509778
BED06376
3E8BF0CF
BEC32D46
BF15B46D
3F20D93B
BEEDC826
3F6AE400
BF0F5D02
3E8CB326
3F9192B7
3F8FA5A4
3E4C6758
3F1A46A9
3E84EEF5
3F74B55A
BF72056A
BF3A5443
3D65FF6F
3F63EC46
400C9E42
3EC5903D
BCB4643F
BEE1760A
BFC0CF6C
3F073491
3F552B16
BFE079C8
C010E625
BFA64FE5
BF61591B
BF8C85B0
BE017B17
BEED5C09
3FA6DC1F
3ED22B6E
3F886204
3EE2E35C
BFE42271
BFB59C94
BFB9C3EA
BFEA2B7B
3F9C91F8
3F0DE935
BE2BB3C5
3EA7D38F
3F5FE41F
BC76702A
BEF6D166
3DB296D6
BF6DF297
BDDEC868
BF17B654
3F8433A5
3DE857D4
3EAE92E6
3F3E885A
3E47F7DF
C0124D99
BF8A06E6
BDCDC808
BEF5E8E2
BF7C20D2
3F43FCB3
400B052C
BF0E539A
BF158C9D
3E8DC1A0
BD28574E
BF0C1D65
BFAE9663
3F47784C
3EC4F091
BF3E2B73
3F689581
3F357332
3F8FDC1C
BEFB3F1C
3F279ADA
BEE704D8
3F54CD15
BF38F1E3
3F49CE8C
BD8D96D1
3E8C208E
3E920A14
BF0D9ED3
3F8B6E97
BF4261E7
3ECB3F94
BECD09E5
3E47DE23
BF4928A9
BEB01A53
BFEAC922
3F80F317
3ED359E2
3F8A0454
BF4B8EDB
BF6F0EC2
BEACF011
BBBDF2BB
3ED10BBD
BFAE0607
BD95BFFA
3FF05C01
BE8169C7
3F1E0213
BED6AAF5
