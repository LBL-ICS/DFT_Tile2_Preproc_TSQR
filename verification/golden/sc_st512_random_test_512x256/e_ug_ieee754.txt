C003D065
BF1B3DE7
3F789656
BC8DFF57
3E750775
3E126CEF
BDAEC896
3E1D31CF
BE135098
3F8B7597
BDC09D7D
3FD91CA9
3D9CDBFC
3E7358A3
BF495057
BF19E29C
C03BA17E
3DF10DDD
3F765203
BF2C017F
BF8CDF1D
BFB2BFC1
3EF196F2
BF477853
3E129303
BF3FD77C
3F251DA3
3DE82E80
BED7A8FE
3FAD786F
3F6FE435
3F7DFB56
3EF274D1
3E2898CA
3F94524C
3EE3103A
BE76F87D
BF772C67
3F010A5A
3E61E594
BF5DA782
BE2D82F2
3E091328
BCAB7B03
3CAD6E74
BEF0803E
3EC0383F
3F418F5C
BF87BBA7
BEFF7B2D
3F457BD4
3F15A6E3
C022459A
BF1C8F3E
BFE5E91E
3F39415A
3FDD1372
3DBFEF03
3FEFFE9D
BF6E9C2C
BE4E66C2
3F64FAAB
BEC495D5
3FCF7AE4
BEC63E54
BFAA5EDB
3DC53675
BF5F19ED
BF1CB4B8
BFB12A1E
3FBEAAD5
BF8DC08B
3E5C930F
BF576FBA
BF6E193B
3F054E27
BF00E2C3
3EE3957A
BEB7F8B4
BE6B5272
BE86E80C
BF743540
BF3FC2A7
3EFBAD85
3EF636FF
401F6544
40072E72
3E9C636F
BFAF502F
BFCB9FFB
40291FE1
BF86F6C9
BEFF50C4
3F78147F
BF168928
3E2251AE
BF309560
BF36DC4E
3F7B7684
3EB4DA21
3E0797BA
3EBDBCE0
BF79D5AF
BF8FD3F4
3F2977EF
3E939B5F
3ED82DCE
3F87733C
C049D4A8
C004B96F
3FA0A5C7
3EF53017
3F227641
3E0E9EC7
BF73AB92
3FAAA698
BF05AD31
BF0EF8E1
BE2F4F5A
3FEA93D4
3ED8B9D3
BF9F9957
3F367358
3EA48231
BEB3A986
BE121B4B
BFCA1C2A
BF1ECE99
3F959C2F
BF4412CA
BF94FB1B
BFA3827F
BEF481EC
3F7A1506
3E9CB44E
BF41D269
BFA3C9E1
3F4F97A0
401917D2
3EB001BA
BF2E3188
3F17FEDB
3F2CA763
BFC40F87
3E0D6916
3F1FFBCB
3E41B788
BFCFF72B
3DEC9DF9
BE73B393
3F82CF49
BF4B27C7
BFF01B86
BEB8C4C9
BF629EA4
BF9DDF78
BE66EEBF
3FFB1ED3
3F5A9248
3F22357F
3F399B0F
3EC4658E
BE976BB1
BE3E6FDD
3F8C55AC
BF1BFD13
3E99D855
3E7B2C9B
3E105BE4
BEE04ABB
BF687777
BF053777
BD513478
BEDF0DA7
BA95E0E0
3E7C1F26
3F5CC394
3F984F46
400B4323
3F0C8B14
4016A803
3E85CBC0
BF424FEC
3F30756C
BF2FD332
3DE61C35
3F917C3C
BE83BE25
BF22E86A
3EF33926
3EE76C36
3FE21246
BF87504B
BF437E08
3F1A81A3
BF1815F7
BF228D27
BEFDD4C2
BE731915
3EDCE595
BFB8E571
BF2F3A70
BEF8807F
BF12CA2D
3F8ADA3C
BF12C350
3F952CC5
BFD199EF
3F537E80
BFB72495
3FF1D964
3F8DF8F7
3F3991DA
BF09ACF4
BEA7328C
BF4DCB46
BFBD1715
BF4443FC
BFD0BA52
C007F7C0
3B6C2E5F
BDBD5952
BEFD179B
BDD1B460
BE893219
BF10F3A3
3E912D68
C0177BD9
BFCB89F7
3ECDF4BC
BFA35567
BF3DAA5D
BF778DB5
BF9E0DDC
BF05A070
3F814829
BFAEC59B
BF3648BC
3DACA92B
3F98402E
BFBE10C7
3F871226
BECA7443
3FE1EA25
3F0C6C14
3F7F56E8
40035973
3D9BA536
BFAD1ED8
3F68291F
BF13E0FD
3FA75A62
BF843EDC
3F37B6F7
3E3E62D7
3F2CDD0C
